charts/app-config/values-integration.yaml

```diff
@@ -236,6 +236,39 @@ govukApplications:
           mongo-3.integration.govuk-internal.digital/content_store_production"
       - name: GOVUK_APP_NAME
         value: content-store
+- name: content-tagger
+  helmValues:
+    dbMigrationEnabled: true
+    workerEnabled: true
+    workerReplicaCount: 1
```

Comment:
optional: you could set this just once at the top of the file (between lines 5 and 6) instead of for every app with workers.

Reply:
that's actually good idea - i will leave that for a seperate PR, as will require cleanup of other apps also which have that in play.